### PR TITLE
[globs::02] Integrate globs with the CLI

### PR DESCRIPTION
**Pull request by kasperski95**
- [x] check settigns globs from `protostar.toml`
- [x] remove old code

EDIT:
- added collecting time feedback
- added activity indicator
- fixed running test runner if no test cases were found

### CODE REVIEW

File: protostar/cli/command.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def directory(arg: str) -> Path:
         is_positional: bool = False
         is_required: bool = False
         is_array: bool = False
-        default: Optional[str] = None
+        default: Any = None
```
Comment (MaksymilianDemitraszek) [inline protostar/cli/command.py:34]: Can we provide Union here instead of `Any`?

**Review by MaksymilianDemitraszek — APPROVED** [review]
Please merge changes

File: protostar/commands/test/test_command.py [hunk 2]
```diff
@@ -88,9 +88,10 @@ async def test(
 
         test_collector_result = TestCollector(
             StarknetCompiler(disable_hint_validation=True, include_paths=include_paths)
-        ).collect(
-            target=target,
-            omit_pattern=omit,
+        ).collect_from_globs(
```
[the remark this answers is not in the document]
Comment (kasperski95) [reply protostar/commands/test/test_command.py:91]: It can. I'll do it in this PR.

File: protostar/cli/command.py (hunk 1 above)
Comment (kasperski95) [reply protostar/cli/command.py:34]: The type of `default` depends on `Argument::type` and `Argument::is_array`. I'd rather mark it as `Any` to tell the developer that he can't rely on the type checker.

**Review by MaksymilianDemitraszek — APPROVED** [review]

**Pull request merged**